packages/@biomejs/biome/README.ru.md

```diff
@@ -0,0 +1,156 @@
+</div>
+
+<br>
+
+**Biome** это производительный инструментарий для веб-проектов, цель которого - предоставить средства разработки для поддержания работоспособности этих проектов.
+
+**Biome это [быстрый форматер](./benchmark#formatting)** для _JavaScript_, _TypeScript_, _JSX_, _JSON_, _CSS_ и _GraphQL_, который достигает **[97% совместимости с _Prettier_](https://console.algora.io/challenges/prettier)**.
+
+**Biome это [производительный линтер](https://github.com/biomejs/biome/tree/main/benchmark#linting)** для _JavaScript_, _TypeScript_, _JSX_, _CSS_ и _GraphQL_, в котором есть **[более 270 правил](https://biomejs.dev/linter/rules/)** из ESLint, typescript-eslint, и [других источников](https://github.com/biomejs/biome/discussions/3).
+Он **предоставляет подробную диагностику с контекстом**, которая поможет вам улучшить ваш код и стать более лучшим программистом!
+
+**Biome** с самого начала был разработан, чтобы использоваться [в интерактивном режиме редактора кода](https://biomejs.dev/guides/integrate-in-editor/).
+Он может форматировать и исправлять некачественный код в процессе его написания.
+
+### Установка
+
+```shell
+npm install --save-dev --save-exact @biomejs/biome
+```
+
+### Использование
+
+```shell
+# форматирование файлов
+npx @biomejs/biome format --write ./src
+
+# линт файлов и применение безопасных изменений
+npx @biomejs/biome lint --write ./src
+
+# форматирование, линт и другие проверки, а также применение безопасных изменений
+npx @biomejs/biome check --write ./src
+
+# форматирование, линт и другие проверки в CI
+npx @biomejs/biome ci ./src
+```
+
+Если вы хотите попробовать Biome, не устанавливая его, используйте [онлайн-песочницу](https://biomejs.dev/playground/), скомпилированную в WebAssembly.
+
+## Документация
+
+Посетите нашу [домашнюю страницу][biomejs], чтобы узнать больше о Biome,
+или сразу перейдите к руководству [Getting Started][getting-started], чтобы начать использовать Biome.
+
+## Больше о Biome
+
+**Biome** имеет разумные настройки по умолчанию и не требует конфигурации.
+
+**Biome** нацелен на поддержку [всех основных языков][language-support] современной веб-разработки.
+
+**Biome** [не требует Node.js](https://biomejs.dev/guides/manual-installation/) для работы.
+
+**Biome** имеет первоклассную поддержку LSP, со сложным синтаксическим анализатором, позволяющим с полной точностью воспроизводить исходный текст, и качественно устраненять ошибки.
```

Comment:
"error recovery" тут поставило меня в тупик, к сожалению, перевёл как понял :(
Спасибо!

Reply:
@ematipico can you help us, please? Here's the context, just in case:

> Biome has first-class LSP support, with a sophisticated parser that represents the source text in full fidelity and top-notch error recovery.

I'd like to know what exactly is the meaning of "first-class" here - something that is implemented very well, at a very high quality level, or some core functionality around which the rest of the system is built?

Reply:
> some core functionality around which the rest of the system is built

This one. When changes/features are made to core, both CLI and LSP benefit from them

Reply:
@denbezrukov предлагаю, в таком случае, опираться на оригинал, и всё же оставить "первоклассную"